src/renderer/atlas/DWriteTextAnalysis.cpp

```diff
@@ -0,0 +1,172 @@
+#include "pch.h"

```

Comment:
note to reviewers: this whole file is just code that was deleted from the `TextAnalyzer` in `AtlasEngine.cpp`